README.md

```diff
@@ -35,30 +52,44 @@ If you are operating Fn, this section is for you.
 * [Log Stores](fn/operate/log-stores.md)
 * [Message Queues](fn/operate/message-queues.md)
 * [Metrics](fn/operate/metrics.md)
-* [Operating on Windows](fn/operate/windows.md)
-* [Private Registries](fn/operate/private_registries.md)
-* [Runtime Options](fn/operate/options.md)
 * [User Interface](fn/operate/ui.md)
 
-### Advanced
+### Learn More
 
 * [Docker Configuration](fn/operate/docker.md)
 * [Extending Fn](fn/operate/extending.md)
 * [Kubernetes Helm Chart for Fn](https://github.com/fnproject/fn-helm/)
+* [Operating on Windows](fn/operate/windows.md)
+* [Private Registries](fn/operate/private_registries.md)
+* [Runtime Options](fn/operate/options.md)
 
 ## For Contributors
 
 If you are working on the Fn Project, want to work on it or are creating extensions, this section is for you.
 
 * [Writing Extensions](fn/develop/extensions.md)
 * [Using Annotations](fn/develop/annotations.md)
+* Join our [Slack Community](http://slack.fnproject.io)
+* See our new [Community Page](https://github.com/fnproject/docs/blob/master/COMMUNITY.md)
+* Learn how to [contribute](CONTRIBUTING.md)
+* Find [issues](https://github.com/fnproject/fn/issues) and become a contributor
+* Join us at one of our [Fn Events](http://events.fnproject.io) or even speak at one!
+* Coming in Q1'19: Regularly scheduled planning meetings for contributing to the Fn Project
```

Comment:
this is just making stale docs, can we remove line 77 until we have meetings we can link to?

Reply:
such lack of faith!! updating...

Reply:
without dates it would be fine :) it's just leaving stale docs to set dates here though...